orc8r/cloud/docker/run.py

```diff
@@ -40,7 +40,7 @@ def main() -> None:
         if args.thanos:
             f.append('thanos')
         file_args = _make_file_args(f)
-        compose_line = 'COMPOSE_FILE={}  {}\n'.format(
```

Comment:
minor nit: This line itself is so confusing, can you simplify this as 
compose_line={docker_files}'.format(docker_files=':'.join(file_args)
compose_line= '\n' + DO_NOT_COMMIT
Might be easy with readability. I couldn't understand what was going on.

Reply:
I understood that in your suggestion it will lost the text "COMPOSE_FILE=" that need to be included in '.env'.